spec/decorators/visit_decorator_spec.rb

```diff
@@ -35,4 +35,76 @@
       end
     end
   end
+
+  describe '#book_to_nomis_opt_in?' do
+    let(:nomis_checker) { instance_double(StaffNomisChecker) }
+
+    before do
+    end
+
+    describe 'when the book to nomis feature is disabled' do
+      before do
+        switch_off :nomis_staff_book_to_nomis_enabled
+      end
+
+      it { expect(subject.book_to_nomis_opt_in?).to eq(false) }
```

Comment:
1) Although this is correct the canonical way to check a boolean in RSpec is:

```ruby
it { expect(subject.book_to_nomis_opt_in?).to be false }
```
2) Also when testing predicate methods, RSpec provide a very nice and readable syntactic sugar API:

```ruby
it { is_expected.not_to be_book_to_nomis_opt_in }
```